cli/.js/src/main/scala/aqua/PlatformUtils.scala

```diff
@@ -0,0 +1,29 @@
+package aqua
+
+import aqua.Utils.logger
+import aqua.js.{Meta, Module}
+import fs2.io.file.Path
+import scribe.Logging
+
+import scala.util.Try
+
+// JS-specific functions
+object PlatformUtils extends Logging {
```

Comment:
If it's js-specific, why `PlatformUtils`? What problem it solves? Is it for builtins? Is it possible to create smth like `BuiltinsSupport` and collect everything needed for builtins support there?